src/widgets/radio.ts

```diff
@@ -0,0 +1,34 @@
+import {Widget} from 'concordialang-ui-core'
+
+import {formatProperties} from './prop'
+
+export class Radio extends Widget {
+	private readonly VALID_PROPERTIES = ['value']
+
+	constructor(props: any, name?: string) {
+		super(props, name || '')
+	}
+
+	public renderToString(): string {
+		const properties = formatProperties(this.props, this.VALID_PROPERTIES)
+		let inputs: String[] = []
+		const label = this.createLabel()
+		const inputName = this.name.toLowerCase()
+
+		if (properties) {
+			for (let value of this.props.value) {
+				let input = `<input type="radio" name="${inputName}" value="${value.toLowerCase()}">${value.trim()}`
+				inputs.push(input)
+			}
+			return `<div>\n${label + inputs.join('\n')}\n</div>`
```

Comment:
I would concern about line breaks here.
I understand that this would be done in another step with prettier or some other tool at the moment the output file is generated.

Reply:
I agree. We can create a new issue to deal with line breaks.

Reply:
Let's keep like this for now and in the near future, we study a better library to generate the HTML files correctly indented.